### PR TITLE
Micu/text under icons in icon bar #270

### PR DESCRIPTION
**Pull request by micuintus**

### CODE REVIEW

**Comment by jmastr** [comment]
Merge it locally to development branch, built it for V-Play Desktop:
```
QQmlApplicationEngine failed to load component
qrc:/qml/harbour-berlin-vegan.qml:179 Type VenueFilterSettings unavailable
qrc:/qml/pages/VenueFilterSettings.qml:107 Type TextSwitch unavailable
qrc:///imports/Sailfish/Silica/TextSwitch.qml:13 Cannot assign to non-existent property "labelFontSize"
```

**Comment by jmastr** [comment]
It is due to V-Play v2.16.1: `labelFontSize` was added with 2.17.0

File: translations/harbour-berlin-vegan-de.ts [hunk 1]
```diff
@@ -302,5 +302,17 @@
         <oldsource>Breakfast</oldsource>
         <translation>Frühstück</translation>
     </message>
+    <message id="id-icon-toolbar-call">
+        <source>CALL</source>
```
Comment (jmastr) [inline translations/harbour-berlin-vegan-de.ts]: Wouldn't it make more sense to use normal casing and use something like `toUppercase()` in the source code? That way the casing is easier to change from the source code...

File: qml/Silica4v-play/TextSwitch.qml [hunk 2]
```diff
@@ -10,4 +10,6 @@ AppCheckBox {
     spacing: BVApp.Theme.paddingLarge
 
     labelColorOff: BVApp.Theme.primaryColor
+    // labelFontSize: BVApp.Theme.fontSizeExtraSmall
```
Comment (jmastr) [inline qml/Silica4v-play/TextSwitch.qml]: Wanna leave this as comment?

**Comment by jmastr** [comment]
@micuintus looks pretty nice. please rebase the branch.

File: qml/Silica4v-play/TextSwitch.qml (hunk 2 above)
Comment (micuintus) [reply qml/Silica4v-play/TextSwitch.qml]: Agreed. Normally I am **really** strongly against commented out code. In this exceptional case, I'd like to keep it in as a reminder.

**Comment by micuintus** [comment]
Rebased.

**Pull request merged**